link/kprobe_test.go

```diff
@@ -337,6 +337,71 @@ func newUpdaterMapProg(t *testing.T, typ ebpf.ProgramType, attach ebpf.AttachTyp
 	return m, p
 }
 
+func newUpdaterMapIncProg(t *testing.T, typ ebpf.ProgramType, attach ebpf.AttachType) (*ebpf.Map, *ebpf.Program) {
```

Comment:
Can you change newUpdaterMapProg instead of copying the implementation? It should be fine to just increment the value always.

Reply:
yep, will change